### PR TITLE
feat: LT-1450 - Paginate and filter goods

### PR DESCRIPTION
**Pull request by Mark-Spiers-kainos**

### CODE REVIEW

File: goods/services.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def get_goods(request, params=None):
     else:
         data = get(request, GOODS_URL)
 
-    return data.json().get("goods"), data.status_code
```
Comment (jfaracik) [inline goods/services.py]: Could you change get_goods to take filter params as opposed to a dict? Makes it more explicit. eg

```
# Organisation
def get_organisations(request, page: int = 1, search_term=None, org_type=None):
    """
    Returns a list of organisations
    :param request: Standard HttpRequest object
    :param page: Returns n page of page results
    :param search_term: Filter by name
    :param org_type: Filter by org type - 'hmrc', 'commercial', 'individual', or an array of it
    """
    data = get(request, ORGANISATIONS_URL + convert_parameters_to_query_params(locals()))
    return data.json()
```

Also can you remove status code

Comment (Mark-Spiers-kainos) [reply goods/services.py]: okay, have updated this to take the filter params as you asked. Going to leave the status_code in for now as I want to tech share a small quality of life piece of code first.

File: templates/goods/goods.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {% extends 'layouts/base.html' %}
 
 {% load svg humanize %}
```
Comment (jfaracik) [inline templates/goods/goods.html]: Move the static inside of this load tag

Comment (Mark-Spiers-kainos) [reply templates/goods/goods.html]: done

**Pull request merged**